state/spaces.go

```diff
@@ -124,6 +124,18 @@ func (s *Space) RenameSpaceOps(toName string) []txn.Op {
 	return renameSpaceOps
 }
 
+// RemoveSpaceOps returns txn.Ops to remove the space
+func (s *Space) RemoveSpaceOps() []txn.Op {
+	return []txn.Op{
+		{
+			C:      volumesC,
```

Comment:
This could go badly for you.

Reply:
this is more of a skeleton and will be changed in the next patch. I think simple removal will not be enough later. You are right!